product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -40,11 +55,15 @@ SELECT bdr.alter_subscription_enable ('bdr_bdrdb_bdrgroup_node2_node1');
 
 ### When to use Parallel Apply
 
-Parallel Apply is always on by default and for most operations, we recommend that it is left on.
+Parallel Apply is always on by default and, for most operations, we recommend that it's left on.
+
+From PGD 5.2, Parallel Apply works with CAMO. It is not compatible with Group Commit or eager replication and should be disabled if Group Commit or eager replication are in use. 
+
+### Monitoring Parallel Apply
 
-### When not to use Parallel Apply
+To support Parallel Apply's deadlock mitigation, PGD 5.2 adds columns to [`bdr.stat_subscription`](/pgd/latest/reference/catalogs-visible#bdrstat_subscription). The new columns are `nprovisional_waits`, `ntuple_waits` and `ncommmit_waits`. The `nprovisional_waits` value reflects the number of operations on the same tuples being performed by concurrent apply transactions. These are provisional waits which aren't actually waiting yet but could. 
 
-As of, and up to at least PGD 5.1, Parallel Apply should not be used with Group Commit, CAMO and eager replication. You should disable Parallel Apply in these scenarios. If you are experiencing a large number of deadlocks, you may also want to disable it.
+If apply operations are affected by the deadlock mitigation measures, they will wait until they can be safely applied and are counted in `ntuple_waits`. Fully applied transactions which waited before being committed are counted in `ncommit_waits`. 
```

Comment:
shouldn't there still be a warning about not using it with Group Commit and Eager and CAMO before 5.1?

Reply:
Fixed further down in the text.